src/ui/primitives/select.rs

```diff
@@ -73,7 +73,7 @@ pub fn select<
 				.height(height)
 				.padding(3)
 				.padding_left(8)
-				.margin(3)
+				.margin(0)
```

Comment:
yeah that was silly of me... why did I do that?

Reply:
do you need `margin` here at all if it is 0?

Reply:
great spot! fixed